app/models/embed/purl.rb

```diff
@@ -208,6 +216,10 @@ def files
         end
       end
 
+      def size
```

Comment:
There are two methods with this name? https://github.com/sul-dlss/sul-embed/pull/1420/files#diff-804f518d2a4799fc0c48e440678aa911984edb69a40efbb0a097a99392b875a8R36

Reply:
NM, this is a Resource class.